spec/models/community_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Community, type: :model do
+  it 'should have valid Factory' do
+    expect(create(:community)).to be_valid
+  end
+
```

Comment:
Should this space be here and if yes, why should it not also be between 12 and 13?

Reply:
fixed